### ISSUE TITLE
bug: dashboard2.0 create route with key-auth plugin, there is a useless setting

### ISSUE THREAD

**Issue by idbeta** [apache/apisix-dashboard#763]
- Why do you submit this issue?
- [ ] Question or discussion
- [x] Bug
- [ ] Requirements
- [ ] Feature or performance improvement
- [ ] Other

![image](https://user-images.githubusercontent.com/7396835/98629039-f767d680-2352-11eb-98e3-307a2eb30936.png)

**Comment by juzhiyuan** [comment]
Is it related to different schema type? cc @liuxiran

**Comment by liuxiran** [comment]
> Is it related to different schema types? cc @liuxiran

Yes, let's add `key-auth` to https://github.com/apache/apisix/issues/2082, and fransfer this issue to APISIX
